src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -226,19 +235,21 @@ def initialize_model_from_path(
     return teacher, model_path, model
 
 
-def initialize_tokenizer_from_path(model_args, model, teacher):
-    tokenizer_src = model_args.tokenizer
-    tokenizer_src = tokenizer_src or get_shared_tokenizer_src(model, teacher)
-    tokenizer = AutoTokenizer.from_pretrained(
-        tokenizer_src,
+def initialize_processor_from_path(
+    model_args: ModelArguments, model: PreTrainedModel, teacher: PreTrainedModel
+) -> Processor:
+    processor_src = model_args.processor
```

Comment:
same, what if a tokenizer is provided?

Reply:
See the newly added [model args handling logic](https://github.com/vllm-project/llm-compressor/pull/955/files#diff-4b4df7cb1f6042ff11af61f87b5d6dffd8e07a7ef182a8aa659566a5ee77fc04R139-R145)